### ISSUE TITLE
error in query10a 

### ISSUE THREAD

**Issue by gregrahn** [gregrahn/tpcds-kit#35]
v2.5.0 introduced query10a but as-is, does not run.  Each branch of the `union all` contains two `where` clauses -- the second one needs to be an `and`.
```
   exists (select *
          from (
             select ws_bill_customer_sk as customer_sk, d_year,d_moy
             from web_sales, date_dim where ws_sold_date_sk = d_date_sk
             where d_year = [YEAR] and
                   d_moy between [MONTH] and [MONTH]+3
             union all
             select cs_ship_customer_sk as customer_sk, d_year, d_moy
             from catalog_sales, date_dim where cs_sold_date_sk = d_date_sk
             where d_year = [YEAR] and
                   d_moy between [MONTH] and [MONTH]+3
	     ) x
            where c.c_customer_sk = customer_sk)
```

**Comment by gregrahn** [comment]
Fixed in master via c3364788e1ad13722d1ce03dff667a9bed6b4eb6

**Comment by gregrahn** [comment]
Fixed in v2.6.0
